lib/restAPIs/userManagement.js

```diff
@@ -19,49 +26,87 @@ module.exports = class RestTopic extends RestUtil {
     if (abbreviationBegins) {
       queryParameter.abbreviationBegins = abbreviationBegins;
     }
-    return this._makeRequest(this._getBase(), queryParameter, 'GET')
-      .then(result => this.throwIfFailed(result, 200));
+    return this._makeRequest(
+      this._getBase(),
+      queryParameter,
+      'GET'
+    ).then(result => this.throwIfFailed(result, 200));
   }
 
-  getUser(userId, fields){
+  /**
+   * @public Get info for a specific user
+   * @param {String} userId - User ID
+   * @param {String} fields - Top level fields to be included in the response
+   */
```

Comment:
- primitive types
- What's the `@return` here?

Reply:
Return Obejct.
Done